tests/security/ima/ima_appraisal_hashes.pm

```diff
@@ -49,7 +53,10 @@ sub run {
 
     replace_grub_cmdline_settings('ima_appraise=fix', '', update_grub => 1);
 
+    # We need re-enable the secureboot after removing "ima_appraise=fix" kernel parameter
     power_action('reboot', textmode => 1);
+    $self->wait_grub(bootloader_time => 200);
+    $self->tianocore_disable_secureboot("re_enable");
```

Comment:
Coding style:
  please using `$self->tianocore_disable_secureboot('re_enable');` for all these kind of lines